src/state/queries/profile.ts

```diff
@@ -25,28 +25,30 @@ import {STALE} from '#/state/queries'
 import {track} from '#/lib/analytics/analytics'
 
 export const RQKEY = (did: string) => ['profile', did]
+export const profileBasicKey = (handle: string) => ['profileBasic', handle]
 export const profilesQueryKey = (handles: string[]) => ['profiles', handles]
 
-export function useProfileQuery({
-  did,
-  staleTime = STALE.SECONDS.FIFTEEN,
-}: {
-  did: string | undefined
-  staleTime?: number
-}) {
-  return useQuery({
-    // WARNING
-    // this staleTime is load-bearing
-    // if you remove it, the UI infinite-loops
-    // -prf
-    staleTime,
+export function useProfileQuery({did}: {did: string | undefined}) {
+  const queryClient = useQueryClient()
+  // TODO Figure out a good staleTime for this. We should refetch on every profile push, because we need to check for
+  // blocks
+  return useQuery<AppBskyActorDefs.ProfileView>({
+    staleTime: STALE.SECONDS.FIFTEEN,
```

Comment:
We should figure out what we want to do with this `staleTime`. Is the removed comment still applicable after these changes?